units/Legion/legcom.lua

```diff
@@ -300,52 +302,51 @@ return {
 					commanders = 1,
 				},
 			},
-			napalmmissile = {
-				areaofeffect = 200,
+			disintegrator = {
```

Comment:
would you be willing to add this normal Dgun to legcomlvl2-4 as well? if not, I can fix it later (after I get some messages about napalm launcher)

Reply:
I'd rather leave the rest in situ for now, this should merge soon and will be an easy fix after